sklearn/mixture/dpgmm.py

```diff
@@ -698,7 +698,8 @@ def __init__(self, n_components=1, covariance_type='diag', alpha=1.0,
             n_components, covariance_type, random_state=random_state,
             tol=tol, verbose=verbose, min_covar=min_covar,
             n_iter=n_iter, params=params, init_params=init_params)
-        self.alpha = float(alpha) / n_components
+        self.alpha = alpha
+        self.alpha_ = float(alpha) / n_components
```

Comment:
this needs to be a property, or set in fit.
If tests pass with setting it in fit, that would be preferable.